fab/fabfile.py

```diff
@@ -367,6 +367,14 @@ def install_packages():
 
 
 @task
+@roles(ROLES_TOUCHFORMS)
+def install_npm_packages():
+    """Install required NPM packages for server"""
+    sudo("cd {} && npm install".format(
+        os.path.join(env.code_root, 'submodules/touchforms-src/touchforms/package.json')))
```

Comment:
This might not always be reliable if there are network connectivity issues, or a new version of a dependency could break previously working code. [see also](https://github.com/dimagi/touchforms/pull/91#issuecomment-96799731)